aiven/client/client.py

```diff
@@ -185,10 +185,10 @@ def build_path(*parts: str) -> str:
 class AivenClient(AivenClientBase):
     """Aiven Client with high-level operations"""
 
-    def get_service_versions(self) -> Mapping:
+    def get_service_versions(self) -> Sequence[Mapping[str, str]]:
         return self.verify(self.get, "/service_versions", result_key="service_versions")
 
-    def get_service_indexes(self, project: str, service: str) -> Mapping:
+    def get_service_indexes(self, project: str, service: str) -> Sequence:
```

Comment:
Can we be more specific with the result type here and in the following functions?